src/backend/marsha/account/urls.py

```diff
@@ -58,7 +59,8 @@
         name="saml_fer_idp_choice",
     ),
     # API
-    path("api/login/", ApiLoginView.as_view(), name="api_login"),
+    path("api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+    path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
```

Comment:
you should also add the url to blacklist a refresh token https://django-rest-framework-simplejwt.readthedocs.io/en/latest/blacklist_app.html